pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -916,17 +959,20 @@ class MMM(
     def channel_contributions_forward_pass(
         self, channel_data: npt.NDArray[np.float64]
     ) -> npt.NDArray[np.float64]:
-        """Evaluate the channel contribution for a given channel data and a fitted model, ie. the forward pass.
-        We return the contribution in the original scale of the target variable.
+        """Evaluate the channel contribution for a given channel data and a fitted model.
+
+        Hence, the forward pass.We return the contribution in the original scale of the target variable.
```

Comment:
what do you mean here?

Reply:
"the forward pass.We" doesn't make sense

Reply:
Thanks! Fixed in https://github.com/pymc-labs/pymc-marketing/pull/951/commits/0af7d7fce1e477b60e7e73105e6932bafba34d2e